sdks/java/io/google-cloud-platform/README.md

```diff
@@ -0,0 +1,70 @@
+# Google Cloud Platform IOs
```

Comment:
We should probably have a README in many places, but we currently don't. Not sure this file alone is not discoverable and useful right now. Perhaps I'd skip it for now, and address that at a later point project-wide.